changelog.md

```diff
@@ -1,5 +1,19 @@
 <!-- Note: this file is auto-generated. See CONTRIBUTING.md for details. -->
 
+### v5.0.0-rc1
+### **Chicken Salad**
+---
+* LLT-3475: Migrate natlab to UniFFI-generated python bindings
+* LLT-5208: Fix boringtun ignoring preshared key change
+* LLT-5215: Temporarily change the underlying UniFFI type for PublicKey and SecretKey
```

Comment:
Is it really temporary?

Reply:
The initial plan was temporary, but this new solution doesn't need to be :+1: